img/CTR_600.png

```diff
@@ -0,0 +1 @@
+/storage/Logo/CTR_600.png
```

Comment:
What's this used for? Again, would prefer to avoid having any CTR specific stuff in the repo if possible.